integrations/sonarqube/.port/resources/blueprints.json

```diff
@@ -124,6 +124,141 @@
     },
     "relations": {}
   },
+  {
+    "identifier": "sonarQubeGAProject",
+    "title": "SonarQube GA Project",
```

Comment:
I see what you're doing here. What I'll prefer is that we have only 1 blueprint for project in the default resource. So let's delete the old SonarQubeProject blueprint, rename this GA project blueprint to SonarQubeProject. This way, when the integration is installed, users see only I SonarQube project blueprint.